core/scripts/ocr2vrf/verify.go

```diff
@@ -105,10 +106,10 @@ func getVRFSignature(e helpers.Environment, beaconAddress string, height, confDe
 	return
 }
 
-func verifyBeaconRandomness(e helpers.Environment, dkgAddress, beaconAddress string, height, confDelay, searchWindow uint64) bool {
+func verifyBeaconRandomness(e helpers.Environment, dkgAddress, coordinatorAddress string, height, confDelay, searchWindow uint64) bool {
```

Comment:
Thinking about this I think it might be useful to have a unit test of the core verification logic (i.e after all the info has been obtained). We can unit-test with real on-chain data.

Reply:
Perhaps for a separate PR though.

Reply:
Oh actually, maybe we should even call this in the integration tests.

Reply:
That way this code gets exercised.

Reply:
CC @jinhoonbang

Reply:
yeah +1 on using `core/scripts` code in the integration tests whereever possible. It would have caught the issue a few days back where a change broke our scripts due to missing `app.Start()`

Reply:
This can be done and is a good idea. However, it would need some logic to be pulled out of the scripts folder, as it's not accessible to our tests in its current structure. I can integrate it after this PR.

Reply:
Yeah we'll have to refactor the verification stuff to live somewhere in the core/services/ocr2/plugins/ocr2vrf package